client/asset/btc/btc.go

```diff
@@ -875,36 +881,48 @@ func newRPCWallet(requester RawRequesterWithContext, cfg *BTCCloneCFG, walletCon
 	return &ExchangeWalletFullNode{btc}, nil
 }
 
-func newUnconnectedWallet(cfg *BTCCloneCFG, walletCfg *WalletConfig) (*baseWallet, error) {
+func validateBtcWalletConf(walletCfg *WalletConfig) error {
+	var err error
 	// If set in the user config, the fallback fee will be in conventional units
 	// per kB, e.g. BTC/kB. Translate that to sats/byte.
-	fallbackFeesPerByte := toSatoshi(walletCfg.FallbackFeeRate / 1000)
-	if fallbackFeesPerByte == 0 {
-		fallbackFeesPerByte = cfg.DefaultFallbackFee
+	walletCfg.FallbackFeeRatePerByte = toSatoshi(walletCfg.FallbackFeeRate / 1000)
+	if walletCfg.FallbackFeeRatePerByte == 0 {
+		err = fmt.Errorf("fallback fee rate limit is smaller than smallest unit(1 sat/byte): %v",
```

Comment:
While true, the form has values of coin/kb. The error is also like this and in scientific notation. So maybe `1e-5 coin/kb`?